packages/frontend-2/components/settings/workspaces/members/UpdateActionsMenu.vue

```diff
@@ -0,0 +1,194 @@
+<template>
```

Comment:
PP: Maybe just `ActionsMenu`, as it contains also non-update actions like delete